src/env.ts

```diff
@@ -41,3 +43,24 @@ export const getGithubEnvVars = (env: Dict<string>) => {
 
   return Object.fromEntries(map);
 };
+
+export const getIPv4s = () => {
+  const nets = networkInterfaces();
+  if (nets === undefined) {
+    return [];
+  }
+
+  const results = [];
+  for (const netIPs of Object.entries(nets)) {
+    if (netIPs[1] === undefined) {
+      continue;
+    }
+
+    for (const ip of netIPs[1]) {
+      if (ip.family == 'IPv4') {
```

Comment:
why not log ipv6?

Reply:
Moved PR to draft till the refinement.
The only thing I know is that it should be just 1 IP address. Maybe IPv6 matches as well.
<img width="452" alt="image" src="https://github.com/user-attachments/assets/afd497d9-14f2-4ca8-9c51-e2ebd3e3257c">
https://segment.com/docs/connections/spec/common/#context

Reply:
Now this is figured out:
<img width="721" alt="image" src="https://github.com/user-attachments/assets/76753c30-73b6-4883-864a-0827fbf9f4d9">